room_allocation.py

```diff
@@ -1,10 +1,13 @@
 import random
```

Comment:
If that models.pyc still exists you shpuld remove it. .pyc files should be in gitignore